crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -201,7 +201,7 @@ fn get_private_key_from_input() -> Result<Felt> {
     parse_input_to_felt(&input)
 }
 
-fn generate_account_name(accounts_file: &Utf8PathBuf) -> Result<String> {
+pub fn generate_account_name(accounts_file: &Utf8PathBuf) -> Result<String> {
```

Comment:
This function is used both in import and creation logic so, let's create a separate file `account.rs` in [`helpers`](https://github.com/foundry-rs/starknet-foundry/tree/master/crates/sncast/src/helpers) and place it there 😃